src/02_spec/07_modules/dem_uart/systemif.rst

```diff
@@ -0,0 +1,65 @@
+System Interface
+----------------
+
+The following signals are required from a bus in order to connect to the DEM.
```

Comment:
"The following signals are required from a bus in order to connect to the DEM."
How about "The DEM-UART module provides a generic bus interface, which can be used by wrapper modules to support actual bus interfaces. We specify a Wishbone wrapper interface below."